config/routes.rb

```diff
@@ -2,5 +2,10 @@
   root 'welcome#index'
   get "users/sign_up" => "users#new"
   post "users/sign_up" => "users#create"
+  get "users/sign_in" => "sessions#new"
+  post "users/sign_in" => "sessions#create"
+  get "users/:id" => "users#show", as: :user
+  delete "logout" => "sessions#destroy"
+  resources :users
```

Comment:
Em đọc mục 4.6 trong hướng dẫn về rails routes https://guides.rubyonrails.org/routing.html#restricting-the-routes-created để restrict những action mà mình muốn dùng thôi nhé